sktime/forecasting/base/adapters/_neuralforecast.py

```diff
@@ -3,13 +3,15 @@
 import abc
 import functools
 import typing
+from inspect import signature
+from logging import warn
```

Comment:
Also, please keep the imports sorted. You can use tools like `isort`.

Optional but highly recommended.